addons/mail/models/discuss/ir_websocket.py

```diff
@@ -22,10 +24,16 @@ def _get_im_status(self, data):
     @add_guest_to_context
     def _build_bus_channel_list(self, channels):
         channels = list(channels)  # do not alter original list
+        channel_ids = list()
```

Comment:
confusing variable name in this context, I'd suggest a longer `discuss_channel_ids` here